src/panels/lovelace/hui-root.js

```diff
@@ -158,24 +174,28 @@ class HUIRoot extends NavigateMixin(EventsMixin(PolymerElement)) {
   }
 
   _selectView(viewIndex) {
-    this._curView = viewIndex;
-
     // Recreate a new element to clear the applied themes.
     const root = this.$.view;
     if (root.lastChild) {
       root.removeChild(root.lastChild);
     }
 
-    const viewConfig = this.config.views[this._curView];
-
     let view;
 
-    if (viewConfig.panel) {
-      view = createCardElement(viewConfig.cards[0]);
+    if (viewIndex === 'unused') {
+      this._curView = -1;
```

Comment:
Why not just change it to unused and keep that assignment at top of the file? That keeps the code a little easier to read.